package.json

```diff
@@ -181,6 +181,7 @@
     "react-addons-test-utils": "15.4.0",
     "react-hot-loader": "1.3.0",
     "react-test-env": "0.2.0",
+    "redux-mock-store": "1.2.1",
```

Comment:
If we decide to keep this, do we need to `make shrinkwrap`? (I'm not sure if it's required for `devDependencies`, but you could try and see if it makes a `diff`erence.)